workflows/qc_only.nf

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I think it's reasonable to leave this here because we now have two smaller execution pathways and it would be good to establish a pattern. I think updating this is in the spirit of what this story requires implementation wise.